HeliPort/Appearance/WiFiMenuItemView.swift

```diff
@@ -93,7 +93,7 @@ class WifiMenuItemView: NSView {
 
     public init(networkInfo: NetworkInfo) {
         self.networkInfo = networkInfo
-        super.init(frame: NSRect(x: 0, y: 0, width: 0, height: menuBarHeight))
+        super.init(frame: NSRect(x: 0, y: 0, width: 286.0, height: menuBarHeight))
```

Comment:
Can you justify the value `286.0`?

Reply:
To resolve these errors during debugging.

DEBUG: Loaded itlwm 2.2.0 as en1
DEBUG: Wi-Fi powered on
flock failed to lock list file (/var/folders/xx/0j9v5xzn7b101rd8x32y843h0000gn/C//com.OpenIntelWireless.HeliPort/com.apple.metal/32023/libraries.list): errno = 35
flock failed to lock list file (/var/folders/xx/0j9v5xzn7b101rd8x32y843h0000gn/C//com.OpenIntelWireless.HeliPort/com.apple.metal/16777237_9765383/functions.list): errno = 35
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bdd60 NSImageView:0x7fd7bf823400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bdf40 NSImageView:0x7fd7bf822180.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031be120 NSImageView:0x7fd7bf822420.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031be030 H:[NSImageView:0x7fd7bf822180]-(12)-[NSImageView:0x7fd7bf822420]   (active)>",
    "<NSLayoutConstraint:0x6000031bde50 H:[NSImageView:0x7fd7bf823400]-(3)-[NSTextField:0x7fd7bf824b20]   (active)>",
    "<NSLayoutConstraint:0x6000031bdef0 H:[NSTextField:0x7fd7bf824b20]-(10)-[NSImageView:0x7fd7bf822180]   (active)>",
    "<NSLayoutConstraint:0x6000031bdd10 NSImageView:0x7fd7bf823400.leading == HeliPort.WifiMenuItemView:0x7fd7bf81e700.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031be080 NSImageView:0x7fd7bf822420.trailing == HeliPort.WifiMenuItemView:0x7fd7bf81e700.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031be5d0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf81e700.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031be120 NSImageView:0x7fd7bf822420.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bdd60 NSImageView:0x7fd7bf823400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bdf40 NSImageView:0x7fd7bf822180.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031be120 NSImageView:0x7fd7bf822420.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031be030 H:[NSImageView:0x7fd7bf822180]-(12)-[NSImageView:0x7fd7bf822420]   (active)>",
    "<NSLayoutConstraint:0x6000031bde50 H:[NSImageView:0x7fd7bf823400]-(3)-[NSTextField:0x7fd7bf824b20]   (active)>",
    "<NSLayoutConstraint:0x6000031bdef0 H:[NSTextField:0x7fd7bf824b20]-(10)-[NSImageView:0x7fd7bf822180]   (active)>",
    "<NSLayoutConstraint:0x6000031bdd10 NSImageView:0x7fd7bf823400.leading == HeliPort.WifiMenuItemView:0x7fd7bf81e700.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031be080 NSImageView:0x7fd7bf822420.trailing == HeliPort.WifiMenuItemView:0x7fd7bf81e700.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031be5d0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf81e700.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031be120 NSImageView:0x7fd7bf822420.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bdd60 NSImageView:0x7fd7bf823400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bdf40 NSImageView:0x7fd7bf822180.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031be030 H:[NSImageView:0x7fd7bf822180]-(12)-[NSImageView:0x7fd7bf822420]   (active)>",
    "<NSLayoutConstraint:0x6000031bde50 H:[NSImageView:0x7fd7bf823400]-(3)-[NSTextField:0x7fd7bf824b20]   (active)>",
    "<NSLayoutConstraint:0x6000031bdef0 H:[NSTextField:0x7fd7bf824b20]-(10)-[NSImageView:0x7fd7bf822180]   (active)>",
    "<NSLayoutConstraint:0x6000031bdd10 NSImageView:0x7fd7bf823400.leading == HeliPort.WifiMenuItemView:0x7fd7bf81e700.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031be080 NSImageView:0x7fd7bf822420.trailing == HeliPort.WifiMenuItemView:0x7fd7bf81e700.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031be5d0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf81e700.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bdf40 NSImageView:0x7fd7bf822180.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bdd60 NSImageView:0x7fd7bf823400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bdf40 NSImageView:0x7fd7bf822180.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031be030 H:[NSImageView:0x7fd7bf822180]-(12)-[NSImageView:0x7fd7bf822420]   (active)>",
    "<NSLayoutConstraint:0x6000031bde50 H:[NSImageView:0x7fd7bf823400]-(3)-[NSTextField:0x7fd7bf824b20]   (active)>",
    "<NSLayoutConstraint:0x6000031bdef0 H:[NSTextField:0x7fd7bf824b20]-(10)-[NSImageView:0x7fd7bf822180]   (active)>",
    "<NSLayoutConstraint:0x6000031bdd10 NSImageView:0x7fd7bf823400.leading == HeliPort.WifiMenuItemView:0x7fd7bf81e700.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031be080 NSImageView:0x7fd7bf822420.trailing == HeliPort.WifiMenuItemView:0x7fd7bf81e700.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031be5d0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf81e700.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bdf40 NSImageView:0x7fd7bf822180.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bdd60 NSImageView:0x7fd7bf823400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031be030 H:[NSImageView:0x7fd7bf822180]-(12)-[NSImageView:0x7fd7bf822420]   (active)>",
    "<NSLayoutConstraint:0x6000031bde50 H:[NSImageView:0x7fd7bf823400]-(3)-[NSTextField:0x7fd7bf824b20]   (active)>",
    "<NSLayoutConstraint:0x6000031bdef0 H:[NSTextField:0x7fd7bf824b20]-(10)-[NSImageView:0x7fd7bf822180]   (active)>",
    "<NSLayoutConstraint:0x6000031bdd10 NSImageView:0x7fd7bf823400.leading == HeliPort.WifiMenuItemView:0x7fd7bf81e700.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031be080 NSImageView:0x7fd7bf822420.trailing == HeliPort.WifiMenuItemView:0x7fd7bf81e700.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031be5d0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf81e700.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bdd60 NSImageView:0x7fd7bf823400.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bdd60 NSImageView:0x7fd7bf823400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031be030 H:[NSImageView:0x7fd7bf822180]-(12)-[NSImageView:0x7fd7bf822420]   (active)>",
    "<NSLayoutConstraint:0x6000031bde50 H:[NSImageView:0x7fd7bf823400]-(3)-[NSTextField:0x7fd7bf824b20]   (active)>",
    "<NSLayoutConstraint:0x6000031bdef0 H:[NSTextField:0x7fd7bf824b20]-(10)-[NSImageView:0x7fd7bf822180]   (active)>",
    "<NSLayoutConstraint:0x6000031bdd10 NSImageView:0x7fd7bf823400.leading == HeliPort.WifiMenuItemView:0x7fd7bf81e700.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031be080 NSImageView:0x7fd7bf822420.trailing == HeliPort.WifiMenuItemView:0x7fd7bf81e700.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031be5d0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf81e700.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bdd60 NSImageView:0x7fd7bf823400.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031be030 H:[NSImageView:0x7fd7bf822180]-(12)-[NSImageView:0x7fd7bf822420]   (active)>",
    "<NSLayoutConstraint:0x6000031bde50 H:[NSImageView:0x7fd7bf823400]-(3)-[NSTextField:0x7fd7bf824b20]   (active)>",
    "<NSLayoutConstraint:0x6000031bdef0 H:[NSTextField:0x7fd7bf824b20]-(10)-[NSImageView:0x7fd7bf822180]   (active)>",
    "<NSLayoutConstraint:0x6000031bdd10 NSImageView:0x7fd7bf823400.leading == HeliPort.WifiMenuItemView:0x7fd7bf81e700.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031be080 NSImageView:0x7fd7bf822420.trailing == HeliPort.WifiMenuItemView:0x7fd7bf81e700.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031be5d0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf81e700.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031be030 H:[NSImageView:0x7fd7bf822180]-(12)-[NSImageView:0x7fd7bf822420]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031be030 H:[NSImageView:0x7fd7bf822180]-(12)-[NSImageView:0x7fd7bf822420]   (active)>",
    "<NSLayoutConstraint:0x6000031bde50 H:[NSImageView:0x7fd7bf823400]-(3)-[NSTextField:0x7fd7bf824b20]   (active)>",
    "<NSLayoutConstraint:0x6000031bdef0 H:[NSTextField:0x7fd7bf824b20]-(10)-[NSImageView:0x7fd7bf822180]   (active)>",
    "<NSLayoutConstraint:0x6000031bdd10 NSImageView:0x7fd7bf823400.leading == HeliPort.WifiMenuItemView:0x7fd7bf81e700.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031be080 NSImageView:0x7fd7bf822420.trailing == HeliPort.WifiMenuItemView:0x7fd7bf81e700.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031be5d0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf81e700.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031be030 H:[NSImageView:0x7fd7bf822180]-(12)-[NSImageView:0x7fd7bf822420]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031be1c0 H:|-(5)-[NSVisualEffectView:0x7fd7bf81bf80](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bf81e700 )>",
    "<NSLayoutConstraint:0x6000031be210 NSVisualEffectView:0x7fd7bf81bf80.right == HeliPort.WifiMenuItemView:0x7fd7bf81e700.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031be5d0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf81e700.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031be210 NSVisualEffectView:0x7fd7bf81bf80.right == HeliPort.WifiMenuItemView:0x7fd7bf81e700.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031be1c0 H:|-(5)-[NSVisualEffectView:0x7fd7bf81bf80](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bf81e700 )>",
    "<NSLayoutConstraint:0x6000031be210 NSVisualEffectView:0x7fd7bf81bf80.right == HeliPort.WifiMenuItemView:0x7fd7bf81e700.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031be5d0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf81e700.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031be210 NSVisualEffectView:0x7fd7bf81bf80.right == HeliPort.WifiMenuItemView:0x7fd7bf81e700.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bf480 NSImageView:0x7fd7bf825d30.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bf660 NSImageView:0x7fd7bf825fd0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bf840 NSImageView:0x7fd7bf827eb0.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031bf750 H:[NSImageView:0x7fd7bf825fd0]-(12)-[NSImageView:0x7fd7bf827eb0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf570 H:[NSImageView:0x7fd7bf825d30]-(3)-[NSTextField:0x7fd7bf828150]   (active)>",
    "<NSLayoutConstraint:0x6000031bf610 H:[NSTextField:0x7fd7bf828150]-(10)-[NSImageView:0x7fd7bf825fd0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf430 NSImageView:0x7fd7bf825d30.leading == HeliPort.WifiMenuItemView:0x7fd7bf826730.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bf7a0 NSImageView:0x7fd7bf827eb0.trailing == HeliPort.WifiMenuItemView:0x7fd7bf826730.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bfcf0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf826730.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bf840 NSImageView:0x7fd7bf827eb0.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bf480 NSImageView:0x7fd7bf825d30.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bf660 NSImageView:0x7fd7bf825fd0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bf840 NSImageView:0x7fd7bf827eb0.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031bf750 H:[NSImageView:0x7fd7bf825fd0]-(12)-[NSImageView:0x7fd7bf827eb0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf570 H:[NSImageView:0x7fd7bf825d30]-(3)-[NSTextField:0x7fd7bf828150]   (active)>",
    "<NSLayoutConstraint:0x6000031bf610 H:[NSTextField:0x7fd7bf828150]-(10)-[NSImageView:0x7fd7bf825fd0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf430 NSImageView:0x7fd7bf825d30.leading == HeliPort.WifiMenuItemView:0x7fd7bf826730.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bf7a0 NSImageView:0x7fd7bf827eb0.trailing == HeliPort.WifiMenuItemView:0x7fd7bf826730.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bfcf0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf826730.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bf840 NSImageView:0x7fd7bf827eb0.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bf480 NSImageView:0x7fd7bf825d30.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bf660 NSImageView:0x7fd7bf825fd0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bf750 H:[NSImageView:0x7fd7bf825fd0]-(12)-[NSImageView:0x7fd7bf827eb0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf570 H:[NSImageView:0x7fd7bf825d30]-(3)-[NSTextField:0x7fd7bf828150]   (active)>",
    "<NSLayoutConstraint:0x6000031bf610 H:[NSTextField:0x7fd7bf828150]-(10)-[NSImageView:0x7fd7bf825fd0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf430 NSImageView:0x7fd7bf825d30.leading == HeliPort.WifiMenuItemView:0x7fd7bf826730.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bf7a0 NSImageView:0x7fd7bf827eb0.trailing == HeliPort.WifiMenuItemView:0x7fd7bf826730.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bfcf0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf826730.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bf660 NSImageView:0x7fd7bf825fd0.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bf480 NSImageView:0x7fd7bf825d30.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bf660 NSImageView:0x7fd7bf825fd0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bf750 H:[NSImageView:0x7fd7bf825fd0]-(12)-[NSImageView:0x7fd7bf827eb0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf570 H:[NSImageView:0x7fd7bf825d30]-(3)-[NSTextField:0x7fd7bf828150]   (active)>",
    "<NSLayoutConstraint:0x6000031bf610 H:[NSTextField:0x7fd7bf828150]-(10)-[NSImageView:0x7fd7bf825fd0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf430 NSImageView:0x7fd7bf825d30.leading == HeliPort.WifiMenuItemView:0x7fd7bf826730.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bf7a0 NSImageView:0x7fd7bf827eb0.trailing == HeliPort.WifiMenuItemView:0x7fd7bf826730.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bfcf0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf826730.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bf660 NSImageView:0x7fd7bf825fd0.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bf480 NSImageView:0x7fd7bf825d30.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bf750 H:[NSImageView:0x7fd7bf825fd0]-(12)-[NSImageView:0x7fd7bf827eb0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf570 H:[NSImageView:0x7fd7bf825d30]-(3)-[NSTextField:0x7fd7bf828150]   (active)>",
    "<NSLayoutConstraint:0x6000031bf610 H:[NSTextField:0x7fd7bf828150]-(10)-[NSImageView:0x7fd7bf825fd0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf430 NSImageView:0x7fd7bf825d30.leading == HeliPort.WifiMenuItemView:0x7fd7bf826730.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bf7a0 NSImageView:0x7fd7bf827eb0.trailing == HeliPort.WifiMenuItemView:0x7fd7bf826730.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bfcf0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf826730.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bf480 NSImageView:0x7fd7bf825d30.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bf480 NSImageView:0x7fd7bf825d30.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bf750 H:[NSImageView:0x7fd7bf825fd0]-(12)-[NSImageView:0x7fd7bf827eb0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf570 H:[NSImageView:0x7fd7bf825d30]-(3)-[NSTextField:0x7fd7bf828150]   (active)>",
    "<NSLayoutConstraint:0x6000031bf610 H:[NSTextField:0x7fd7bf828150]-(10)-[NSImageView:0x7fd7bf825fd0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf430 NSImageView:0x7fd7bf825d30.leading == HeliPort.WifiMenuItemView:0x7fd7bf826730.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bf7a0 NSImageView:0x7fd7bf827eb0.trailing == HeliPort.WifiMenuItemView:0x7fd7bf826730.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bfcf0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf826730.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bf480 NSImageView:0x7fd7bf825d30.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bf750 H:[NSImageView:0x7fd7bf825fd0]-(12)-[NSImageView:0x7fd7bf827eb0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf570 H:[NSImageView:0x7fd7bf825d30]-(3)-[NSTextField:0x7fd7bf828150]   (active)>",
    "<NSLayoutConstraint:0x6000031bf610 H:[NSTextField:0x7fd7bf828150]-(10)-[NSImageView:0x7fd7bf825fd0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf430 NSImageView:0x7fd7bf825d30.leading == HeliPort.WifiMenuItemView:0x7fd7bf826730.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bf7a0 NSImageView:0x7fd7bf827eb0.trailing == HeliPort.WifiMenuItemView:0x7fd7bf826730.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bfcf0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf826730.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bf750 H:[NSImageView:0x7fd7bf825fd0]-(12)-[NSImageView:0x7fd7bf827eb0]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bf750 H:[NSImageView:0x7fd7bf825fd0]-(12)-[NSImageView:0x7fd7bf827eb0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf570 H:[NSImageView:0x7fd7bf825d30]-(3)-[NSTextField:0x7fd7bf828150]   (active)>",
    "<NSLayoutConstraint:0x6000031bf610 H:[NSTextField:0x7fd7bf828150]-(10)-[NSImageView:0x7fd7bf825fd0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf430 NSImageView:0x7fd7bf825d30.leading == HeliPort.WifiMenuItemView:0x7fd7bf826730.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bf7a0 NSImageView:0x7fd7bf827eb0.trailing == HeliPort.WifiMenuItemView:0x7fd7bf826730.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bfcf0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf826730.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bf750 H:[NSImageView:0x7fd7bf825fd0]-(12)-[NSImageView:0x7fd7bf827eb0]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bf8e0 H:|-(5)-[NSVisualEffectView:0x7fd7bf825a90](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bf826730 )>",
    "<NSLayoutConstraint:0x6000031bf930 NSVisualEffectView:0x7fd7bf825a90.right == HeliPort.WifiMenuItemView:0x7fd7bf826730.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bfcf0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf826730.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bf930 NSVisualEffectView:0x7fd7bf825a90.right == HeliPort.WifiMenuItemView:0x7fd7bf826730.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bf8e0 H:|-(5)-[NSVisualEffectView:0x7fd7bf825a90](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bf826730 )>",
    "<NSLayoutConstraint:0x6000031bf930 NSVisualEffectView:0x7fd7bf825a90.right == HeliPort.WifiMenuItemView:0x7fd7bf826730.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bfcf0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bf826730.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bf930 NSVisualEffectView:0x7fd7bf825a90.right == HeliPort.WifiMenuItemView:0x7fd7bf826730.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031c9590 NSImageView:0x7fd7bef276a0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c9770 NSImageView:0x7fd7bef27ab0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031c9950 NSImageView:0x7fd7bef27d50.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031c9860 H:[NSImageView:0x7fd7bef27ab0]-(12)-[NSImageView:0x7fd7bef27d50]   (active)>",
    "<NSLayoutConstraint:0x6000031c9680 H:[NSImageView:0x7fd7bef276a0]-(3)-[NSTextField:0x7fd7bef27ff0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9720 H:[NSTextField:0x7fd7bef27ff0]-(10)-[NSImageView:0x7fd7bef27ab0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9540 NSImageView:0x7fd7bef276a0.leading == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c98b0 NSImageView:0x7fd7bef27d50.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c9e00 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031c9950 NSImageView:0x7fd7bef27d50.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031c9590 NSImageView:0x7fd7bef276a0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c9770 NSImageView:0x7fd7bef27ab0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031c9950 NSImageView:0x7fd7bef27d50.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031c9860 H:[NSImageView:0x7fd7bef27ab0]-(12)-[NSImageView:0x7fd7bef27d50]   (active)>",
    "<NSLayoutConstraint:0x6000031c9680 H:[NSImageView:0x7fd7bef276a0]-(3)-[NSTextField:0x7fd7bef27ff0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9720 H:[NSTextField:0x7fd7bef27ff0]-(10)-[NSImageView:0x7fd7bef27ab0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9540 NSImageView:0x7fd7bef276a0.leading == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c98b0 NSImageView:0x7fd7bef27d50.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c9e00 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031c9950 NSImageView:0x7fd7bef27d50.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031c9590 NSImageView:0x7fd7bef276a0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c9770 NSImageView:0x7fd7bef27ab0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031c9860 H:[NSImageView:0x7fd7bef27ab0]-(12)-[NSImageView:0x7fd7bef27d50]   (active)>",
    "<NSLayoutConstraint:0x6000031c9680 H:[NSImageView:0x7fd7bef276a0]-(3)-[NSTextField:0x7fd7bef27ff0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9720 H:[NSTextField:0x7fd7bef27ff0]-(10)-[NSImageView:0x7fd7bef27ab0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9540 NSImageView:0x7fd7bef276a0.leading == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c98b0 NSImageView:0x7fd7bef27d50.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c9e00 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031c9770 NSImageView:0x7fd7bef27ab0.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031c9590 NSImageView:0x7fd7bef276a0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c9770 NSImageView:0x7fd7bef27ab0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031c9860 H:[NSImageView:0x7fd7bef27ab0]-(12)-[NSImageView:0x7fd7bef27d50]   (active)>",
    "<NSLayoutConstraint:0x6000031c9680 H:[NSImageView:0x7fd7bef276a0]-(3)-[NSTextField:0x7fd7bef27ff0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9720 H:[NSTextField:0x7fd7bef27ff0]-(10)-[NSImageView:0x7fd7bef27ab0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9540 NSImageView:0x7fd7bef276a0.leading == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c98b0 NSImageView:0x7fd7bef27d50.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c9e00 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031c9770 NSImageView:0x7fd7bef27ab0.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031c9590 NSImageView:0x7fd7bef276a0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c9860 H:[NSImageView:0x7fd7bef27ab0]-(12)-[NSImageView:0x7fd7bef27d50]   (active)>",
    "<NSLayoutConstraint:0x6000031c9680 H:[NSImageView:0x7fd7bef276a0]-(3)-[NSTextField:0x7fd7bef27ff0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9720 H:[NSTextField:0x7fd7bef27ff0]-(10)-[NSImageView:0x7fd7bef27ab0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9540 NSImageView:0x7fd7bef276a0.leading == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c98b0 NSImageView:0x7fd7bef27d50.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c9e00 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031c9590 NSImageView:0x7fd7bef276a0.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031c9590 NSImageView:0x7fd7bef276a0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c9860 H:[NSImageView:0x7fd7bef27ab0]-(12)-[NSImageView:0x7fd7bef27d50]   (active)>",
    "<NSLayoutConstraint:0x6000031c9680 H:[NSImageView:0x7fd7bef276a0]-(3)-[NSTextField:0x7fd7bef27ff0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9720 H:[NSTextField:0x7fd7bef27ff0]-(10)-[NSImageView:0x7fd7bef27ab0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9540 NSImageView:0x7fd7bef276a0.leading == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c98b0 NSImageView:0x7fd7bef27d50.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c9e00 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031c9590 NSImageView:0x7fd7bef276a0.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031c9860 H:[NSImageView:0x7fd7bef27ab0]-(12)-[NSImageView:0x7fd7bef27d50]   (active)>",
    "<NSLayoutConstraint:0x6000031c9680 H:[NSImageView:0x7fd7bef276a0]-(3)-[NSTextField:0x7fd7bef27ff0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9720 H:[NSTextField:0x7fd7bef27ff0]-(10)-[NSImageView:0x7fd7bef27ab0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9540 NSImageView:0x7fd7bef276a0.leading == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c98b0 NSImageView:0x7fd7bef27d50.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c9e00 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031c9860 H:[NSImageView:0x7fd7bef27ab0]-(12)-[NSImageView:0x7fd7bef27d50]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031c9860 H:[NSImageView:0x7fd7bef27ab0]-(12)-[NSImageView:0x7fd7bef27d50]   (active)>",
    "<NSLayoutConstraint:0x6000031c9680 H:[NSImageView:0x7fd7bef276a0]-(3)-[NSTextField:0x7fd7bef27ff0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9720 H:[NSTextField:0x7fd7bef27ff0]-(10)-[NSImageView:0x7fd7bef27ab0]   (active)>",
    "<NSLayoutConstraint:0x6000031c9540 NSImageView:0x7fd7bef276a0.leading == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c98b0 NSImageView:0x7fd7bef27d50.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c9e00 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031c9860 H:[NSImageView:0x7fd7bef27ab0]-(12)-[NSImageView:0x7fd7bef27d50]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031c99f0 H:|-(5)-[NSVisualEffectView:0x7fd7bef27400](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bef1a3d0 )>",
    "<NSLayoutConstraint:0x6000031c9a40 NSVisualEffectView:0x7fd7bef27400.right == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c9e00 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031c9a40 NSVisualEffectView:0x7fd7bef27400.right == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031c99f0 H:|-(5)-[NSVisualEffectView:0x7fd7bef27400](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bef1a3d0 )>",
    "<NSLayoutConstraint:0x6000031c9a40 NSVisualEffectView:0x7fd7bef27400.right == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c9e00 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031c9a40 NSVisualEffectView:0x7fd7bef27400.right == HeliPort.WifiMenuItemView:0x7fd7bef1a3d0.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b6120 NSImageView:0x7fd7bfa1a730.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b6300 NSImageView:0x7fd7bfa18790.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b64e0 NSImageView:0x7fd7bfa197d0.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031b63f0 H:[NSImageView:0x7fd7bfa18790]-(12)-[NSImageView:0x7fd7bfa197d0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6210 H:[NSImageView:0x7fd7bfa1a730]-(3)-[NSTextField:0x7fd7bfa19a70]   (active)>",
    "<NSLayoutConstraint:0x6000031b62b0 H:[NSTextField:0x7fd7bfa19a70]-(10)-[NSImageView:0x7fd7bfa18790]   (active)>",
    "<NSLayoutConstraint:0x6000031b60d0 NSImageView:0x7fd7bfa1a730.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b6440 NSImageView:0x7fd7bfa197d0.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b6990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1a210.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b64e0 NSImageView:0x7fd7bfa197d0.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b6120 NSImageView:0x7fd7bfa1a730.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b6300 NSImageView:0x7fd7bfa18790.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b64e0 NSImageView:0x7fd7bfa197d0.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031b63f0 H:[NSImageView:0x7fd7bfa18790]-(12)-[NSImageView:0x7fd7bfa197d0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6210 H:[NSImageView:0x7fd7bfa1a730]-(3)-[NSTextField:0x7fd7bfa19a70]   (active)>",
    "<NSLayoutConstraint:0x6000031b62b0 H:[NSTextField:0x7fd7bfa19a70]-(10)-[NSImageView:0x7fd7bfa18790]   (active)>",
    "<NSLayoutConstraint:0x6000031b60d0 NSImageView:0x7fd7bfa1a730.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b6440 NSImageView:0x7fd7bfa197d0.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b6990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1a210.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b64e0 NSImageView:0x7fd7bfa197d0.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b6120 NSImageView:0x7fd7bfa1a730.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b6300 NSImageView:0x7fd7bfa18790.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b63f0 H:[NSImageView:0x7fd7bfa18790]-(12)-[NSImageView:0x7fd7bfa197d0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6210 H:[NSImageView:0x7fd7bfa1a730]-(3)-[NSTextField:0x7fd7bfa19a70]   (active)>",
    "<NSLayoutConstraint:0x6000031b62b0 H:[NSTextField:0x7fd7bfa19a70]-(10)-[NSImageView:0x7fd7bfa18790]   (active)>",
    "<NSLayoutConstraint:0x6000031b60d0 NSImageView:0x7fd7bfa1a730.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b6440 NSImageView:0x7fd7bfa197d0.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b6990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1a210.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b6300 NSImageView:0x7fd7bfa18790.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b6120 NSImageView:0x7fd7bfa1a730.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b6300 NSImageView:0x7fd7bfa18790.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b63f0 H:[NSImageView:0x7fd7bfa18790]-(12)-[NSImageView:0x7fd7bfa197d0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6210 H:[NSImageView:0x7fd7bfa1a730]-(3)-[NSTextField:0x7fd7bfa19a70]   (active)>",
    "<NSLayoutConstraint:0x6000031b62b0 H:[NSTextField:0x7fd7bfa19a70]-(10)-[NSImageView:0x7fd7bfa18790]   (active)>",
    "<NSLayoutConstraint:0x6000031b60d0 NSImageView:0x7fd7bfa1a730.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b6440 NSImageView:0x7fd7bfa197d0.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b6990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1a210.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b6300 NSImageView:0x7fd7bfa18790.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b6120 NSImageView:0x7fd7bfa1a730.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b63f0 H:[NSImageView:0x7fd7bfa18790]-(12)-[NSImageView:0x7fd7bfa197d0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6210 H:[NSImageView:0x7fd7bfa1a730]-(3)-[NSTextField:0x7fd7bfa19a70]   (active)>",
    "<NSLayoutConstraint:0x6000031b62b0 H:[NSTextField:0x7fd7bfa19a70]-(10)-[NSImageView:0x7fd7bfa18790]   (active)>",
    "<NSLayoutConstraint:0x6000031b60d0 NSImageView:0x7fd7bfa1a730.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b6440 NSImageView:0x7fd7bfa197d0.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b6990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1a210.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b6120 NSImageView:0x7fd7bfa1a730.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b6120 NSImageView:0x7fd7bfa1a730.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b63f0 H:[NSImageView:0x7fd7bfa18790]-(12)-[NSImageView:0x7fd7bfa197d0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6210 H:[NSImageView:0x7fd7bfa1a730]-(3)-[NSTextField:0x7fd7bfa19a70]   (active)>",
    "<NSLayoutConstraint:0x6000031b62b0 H:[NSTextField:0x7fd7bfa19a70]-(10)-[NSImageView:0x7fd7bfa18790]   (active)>",
    "<NSLayoutConstraint:0x6000031b60d0 NSImageView:0x7fd7bfa1a730.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b6440 NSImageView:0x7fd7bfa197d0.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b6990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1a210.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b6120 NSImageView:0x7fd7bfa1a730.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b63f0 H:[NSImageView:0x7fd7bfa18790]-(12)-[NSImageView:0x7fd7bfa197d0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6210 H:[NSImageView:0x7fd7bfa1a730]-(3)-[NSTextField:0x7fd7bfa19a70]   (active)>",
    "<NSLayoutConstraint:0x6000031b62b0 H:[NSTextField:0x7fd7bfa19a70]-(10)-[NSImageView:0x7fd7bfa18790]   (active)>",
    "<NSLayoutConstraint:0x6000031b60d0 NSImageView:0x7fd7bfa1a730.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b6440 NSImageView:0x7fd7bfa197d0.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b6990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1a210.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b63f0 H:[NSImageView:0x7fd7bfa18790]-(12)-[NSImageView:0x7fd7bfa197d0]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b63f0 H:[NSImageView:0x7fd7bfa18790]-(12)-[NSImageView:0x7fd7bfa197d0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6210 H:[NSImageView:0x7fd7bfa1a730]-(3)-[NSTextField:0x7fd7bfa19a70]   (active)>",
    "<NSLayoutConstraint:0x6000031b62b0 H:[NSTextField:0x7fd7bfa19a70]-(10)-[NSImageView:0x7fd7bfa18790]   (active)>",
    "<NSLayoutConstraint:0x6000031b60d0 NSImageView:0x7fd7bfa1a730.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b6440 NSImageView:0x7fd7bfa197d0.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b6990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1a210.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b63f0 H:[NSImageView:0x7fd7bfa18790]-(12)-[NSImageView:0x7fd7bfa197d0]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b6580 H:|-(5)-[NSVisualEffectView:0x7fd7bfa1a490](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfa1a210 )>",
    "<NSLayoutConstraint:0x6000031b65d0 NSVisualEffectView:0x7fd7bfa1a490.right == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b6990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1a210.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b65d0 NSVisualEffectView:0x7fd7bfa1a490.right == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b6580 H:|-(5)-[NSVisualEffectView:0x7fd7bfa1a490](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfa1a210 )>",
    "<NSLayoutConstraint:0x6000031b65d0 NSVisualEffectView:0x7fd7bfa1a490.right == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b6990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1a210.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b65d0 NSVisualEffectView:0x7fd7bfa1a490.right == HeliPort.WifiMenuItemView:0x7fd7bfa1a210.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b7890 NSImageView:0x7fd7bfa1b8d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b7a70 NSImageView:0x7fd7bfa1bb70.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b7c50 NSImageView:0x7fd7bfa1be10.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031b7b60 H:[NSImageView:0x7fd7bfa1bb70]-(12)-[NSImageView:0x7fd7bfa1be10]   (active)>",
    "<NSLayoutConstraint:0x6000031b7980 H:[NSImageView:0x7fd7bfa1b8d0]-(3)-[NSTextField:0x7fd7bfa1da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7a20 H:[NSTextField:0x7fd7bfa1da70]-(10)-[NSImageView:0x7fd7bfa1bb70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7840 NSImageView:0x7fd7bfa1b8d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7bb0 NSImageView:0x7fd7bfa1be10.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c40a0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b7c50 NSImageView:0x7fd7bfa1be10.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b7890 NSImageView:0x7fd7bfa1b8d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b7a70 NSImageView:0x7fd7bfa1bb70.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b7c50 NSImageView:0x7fd7bfa1be10.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031b7b60 H:[NSImageView:0x7fd7bfa1bb70]-(12)-[NSImageView:0x7fd7bfa1be10]   (active)>",
    "<NSLayoutConstraint:0x6000031b7980 H:[NSImageView:0x7fd7bfa1b8d0]-(3)-[NSTextField:0x7fd7bfa1da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7a20 H:[NSTextField:0x7fd7bfa1da70]-(10)-[NSImageView:0x7fd7bfa1bb70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7840 NSImageView:0x7fd7bfa1b8d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7bb0 NSImageView:0x7fd7bfa1be10.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c40a0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b7c50 NSImageView:0x7fd7bfa1be10.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b7890 NSImageView:0x7fd7bfa1b8d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b7a70 NSImageView:0x7fd7bfa1bb70.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b7b60 H:[NSImageView:0x7fd7bfa1bb70]-(12)-[NSImageView:0x7fd7bfa1be10]   (active)>",
    "<NSLayoutConstraint:0x6000031b7980 H:[NSImageView:0x7fd7bfa1b8d0]-(3)-[NSTextField:0x7fd7bfa1da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7a20 H:[NSTextField:0x7fd7bfa1da70]-(10)-[NSImageView:0x7fd7bfa1bb70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7840 NSImageView:0x7fd7bfa1b8d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7bb0 NSImageView:0x7fd7bfa1be10.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c40a0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b7a70 NSImageView:0x7fd7bfa1bb70.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b7890 NSImageView:0x7fd7bfa1b8d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b7a70 NSImageView:0x7fd7bfa1bb70.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b7b60 H:[NSImageView:0x7fd7bfa1bb70]-(12)-[NSImageView:0x7fd7bfa1be10]   (active)>",
    "<NSLayoutConstraint:0x6000031b7980 H:[NSImageView:0x7fd7bfa1b8d0]-(3)-[NSTextField:0x7fd7bfa1da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7a20 H:[NSTextField:0x7fd7bfa1da70]-(10)-[NSImageView:0x7fd7bfa1bb70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7840 NSImageView:0x7fd7bfa1b8d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7bb0 NSImageView:0x7fd7bfa1be10.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c40a0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b7a70 NSImageView:0x7fd7bfa1bb70.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b7890 NSImageView:0x7fd7bfa1b8d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b7b60 H:[NSImageView:0x7fd7bfa1bb70]-(12)-[NSImageView:0x7fd7bfa1be10]   (active)>",
    "<NSLayoutConstraint:0x6000031b7980 H:[NSImageView:0x7fd7bfa1b8d0]-(3)-[NSTextField:0x7fd7bfa1da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7a20 H:[NSTextField:0x7fd7bfa1da70]-(10)-[NSImageView:0x7fd7bfa1bb70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7840 NSImageView:0x7fd7bfa1b8d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7bb0 NSImageView:0x7fd7bfa1be10.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c40a0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b7890 NSImageView:0x7fd7bfa1b8d0.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b7890 NSImageView:0x7fd7bfa1b8d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b7b60 H:[NSImageView:0x7fd7bfa1bb70]-(12)-[NSImageView:0x7fd7bfa1be10]   (active)>",
    "<NSLayoutConstraint:0x6000031b7980 H:[NSImageView:0x7fd7bfa1b8d0]-(3)-[NSTextField:0x7fd7bfa1da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7a20 H:[NSTextField:0x7fd7bfa1da70]-(10)-[NSImageView:0x7fd7bfa1bb70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7840 NSImageView:0x7fd7bfa1b8d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7bb0 NSImageView:0x7fd7bfa1be10.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c40a0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b7890 NSImageView:0x7fd7bfa1b8d0.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b7b60 H:[NSImageView:0x7fd7bfa1bb70]-(12)-[NSImageView:0x7fd7bfa1be10]   (active)>",
    "<NSLayoutConstraint:0x6000031b7980 H:[NSImageView:0x7fd7bfa1b8d0]-(3)-[NSTextField:0x7fd7bfa1da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7a20 H:[NSTextField:0x7fd7bfa1da70]-(10)-[NSImageView:0x7fd7bfa1bb70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7840 NSImageView:0x7fd7bfa1b8d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7bb0 NSImageView:0x7fd7bfa1be10.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c40a0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b7b60 H:[NSImageView:0x7fd7bfa1bb70]-(12)-[NSImageView:0x7fd7bfa1be10]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b7b60 H:[NSImageView:0x7fd7bfa1bb70]-(12)-[NSImageView:0x7fd7bfa1be10]   (active)>",
    "<NSLayoutConstraint:0x6000031b7980 H:[NSImageView:0x7fd7bfa1b8d0]-(3)-[NSTextField:0x7fd7bfa1da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7a20 H:[NSTextField:0x7fd7bfa1da70]-(10)-[NSImageView:0x7fd7bfa1bb70]   (active)>",
    "<NSLayoutConstraint:0x6000031b7840 NSImageView:0x7fd7bfa1b8d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7bb0 NSImageView:0x7fd7bfa1be10.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c40a0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b7b60 H:[NSImageView:0x7fd7bfa1bb70]-(12)-[NSImageView:0x7fd7bfa1be10]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b7cf0 H:|-(5)-[NSVisualEffectView:0x7fd7bfa1d530](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0 )>",
    "<NSLayoutConstraint:0x6000031b7d40 NSVisualEffectView:0x7fd7bfa1d530.right == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c40a0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b7d40 NSVisualEffectView:0x7fd7bfa1d530.right == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b7cf0 H:|-(5)-[NSVisualEffectView:0x7fd7bfa1d530](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0 )>",
    "<NSLayoutConstraint:0x6000031b7d40 NSVisualEffectView:0x7fd7bfa1d530.right == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c40a0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b7d40 NSVisualEffectView:0x7fd7bfa1d530.right == HeliPort.WifiMenuItemView:0x7fd7bfa1c6c0.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031c45f0 NSImageView:0x7fd7bef2a320.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c48c0 NSImageView:0x7fd7bfa1e850.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031c4aa0 NSImageView:0x7fd7bfa1eaf0.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031c49b0 H:[NSImageView:0x7fd7bfa1e850]-(12)-[NSImageView:0x7fd7bfa1eaf0]   (active)>",
    "<NSLayoutConstraint:0x6000031c47d0 H:[NSImageView:0x7fd7bef2a320]-(3)-[NSTextField:0x7fd7bfa1ed90]   (active)>",
    "<NSLayoutConstraint:0x6000031c4870 H:[NSTextField:0x7fd7bfa1ed90]-(10)-[NSImageView:0x7fd7bfa1e850]   (active)>",
    "<NSLayoutConstraint:0x6000031c4690 NSImageView:0x7fd7bef2a320.leading == HeliPort.WifiMenuItemView:0x7fd7bef29e00.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c4a00 NSImageView:0x7fd7bfa1eaf0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef29e00.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c4f50 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef29e00.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031c4aa0 NSImageView:0x7fd7bfa1eaf0.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031c45f0 NSImageView:0x7fd7bef2a320.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c48c0 NSImageView:0x7fd7bfa1e850.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031c4aa0 NSImageView:0x7fd7bfa1eaf0.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031c49b0 H:[NSImageView:0x7fd7bfa1e850]-(12)-[NSImageView:0x7fd7bfa1eaf0]   (active)>",
    "<NSLayoutConstraint:0x6000031c47d0 H:[NSImageView:0x7fd7bef2a320]-(3)-[NSTextField:0x7fd7bfa1ed90]   (active)>",
    "<NSLayoutConstraint:0x6000031c4870 H:[NSTextField:0x7fd7bfa1ed90]-(10)-[NSImageView:0x7fd7bfa1e850]   (active)>",
    "<NSLayoutConstraint:0x6000031c4690 NSImageView:0x7fd7bef2a320.leading == HeliPort.WifiMenuItemView:0x7fd7bef29e00.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c4a00 NSImageView:0x7fd7bfa1eaf0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef29e00.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c4f50 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef29e00.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031c4aa0 NSImageView:0x7fd7bfa1eaf0.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031c45f0 NSImageView:0x7fd7bef2a320.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c48c0 NSImageView:0x7fd7bfa1e850.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031c49b0 H:[NSImageView:0x7fd7bfa1e850]-(12)-[NSImageView:0x7fd7bfa1eaf0]   (active)>",
    "<NSLayoutConstraint:0x6000031c47d0 H:[NSImageView:0x7fd7bef2a320]-(3)-[NSTextField:0x7fd7bfa1ed90]   (active)>",
    "<NSLayoutConstraint:0x6000031c4870 H:[NSTextField:0x7fd7bfa1ed90]-(10)-[NSImageView:0x7fd7bfa1e850]   (active)>",
    "<NSLayoutConstraint:0x6000031c4690 NSImageView:0x7fd7bef2a320.leading == HeliPort.WifiMenuItemView:0x7fd7bef29e00.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c4a00 NSImageView:0x7fd7bfa1eaf0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef29e00.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c4f50 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef29e00.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031c48c0 NSImageView:0x7fd7bfa1e850.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031c45f0 NSImageView:0x7fd7bef2a320.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c48c0 NSImageView:0x7fd7bfa1e850.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031c49b0 H:[NSImageView:0x7fd7bfa1e850]-(12)-[NSImageView:0x7fd7bfa1eaf0]   (active)>",
    "<NSLayoutConstraint:0x6000031c47d0 H:[NSImageView:0x7fd7bef2a320]-(3)-[NSTextField:0x7fd7bfa1ed90]   (active)>",
    "<NSLayoutConstraint:0x6000031c4870 H:[NSTextField:0x7fd7bfa1ed90]-(10)-[NSImageView:0x7fd7bfa1e850]   (active)>",
    "<NSLayoutConstraint:0x6000031c4690 NSImageView:0x7fd7bef2a320.leading == HeliPort.WifiMenuItemView:0x7fd7bef29e00.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c4a00 NSImageView:0x7fd7bfa1eaf0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef29e00.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c4f50 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef29e00.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031c48c0 NSImageView:0x7fd7bfa1e850.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031c45f0 NSImageView:0x7fd7bef2a320.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c49b0 H:[NSImageView:0x7fd7bfa1e850]-(12)-[NSImageView:0x7fd7bfa1eaf0]   (active)>",
    "<NSLayoutConstraint:0x6000031c47d0 H:[NSImageView:0x7fd7bef2a320]-(3)-[NSTextField:0x7fd7bfa1ed90]   (active)>",
    "<NSLayoutConstraint:0x6000031c4870 H:[NSTextField:0x7fd7bfa1ed90]-(10)-[NSImageView:0x7fd7bfa1e850]   (active)>",
    "<NSLayoutConstraint:0x6000031c4690 NSImageView:0x7fd7bef2a320.leading == HeliPort.WifiMenuItemView:0x7fd7bef29e00.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c4a00 NSImageView:0x7fd7bfa1eaf0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef29e00.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c4f50 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef29e00.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031c45f0 NSImageView:0x7fd7bef2a320.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031c45f0 NSImageView:0x7fd7bef2a320.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031c49b0 H:[NSImageView:0x7fd7bfa1e850]-(12)-[NSImageView:0x7fd7bfa1eaf0]   (active)>",
    "<NSLayoutConstraint:0x6000031c47d0 H:[NSImageView:0x7fd7bef2a320]-(3)-[NSTextField:0x7fd7bfa1ed90]   (active)>",
    "<NSLayoutConstraint:0x6000031c4870 H:[NSTextField:0x7fd7bfa1ed90]-(10)-[NSImageView:0x7fd7bfa1e850]   (active)>",
    "<NSLayoutConstraint:0x6000031c4690 NSImageView:0x7fd7bef2a320.leading == HeliPort.WifiMenuItemView:0x7fd7bef29e00.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c4a00 NSImageView:0x7fd7bfa1eaf0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef29e00.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c4f50 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef29e00.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031c45f0 NSImageView:0x7fd7bef2a320.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031c49b0 H:[NSImageView:0x7fd7bfa1e850]-(12)-[NSImageView:0x7fd7bfa1eaf0]   (active)>",
    "<NSLayoutConstraint:0x6000031c47d0 H:[NSImageView:0x7fd7bef2a320]-(3)-[NSTextField:0x7fd7bfa1ed90]   (active)>",
    "<NSLayoutConstraint:0x6000031c4870 H:[NSTextField:0x7fd7bfa1ed90]-(10)-[NSImageView:0x7fd7bfa1e850]   (active)>",
    "<NSLayoutConstraint:0x6000031c4690 NSImageView:0x7fd7bef2a320.leading == HeliPort.WifiMenuItemView:0x7fd7bef29e00.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c4a00 NSImageView:0x7fd7bfa1eaf0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef29e00.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c4f50 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef29e00.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031c49b0 H:[NSImageView:0x7fd7bfa1e850]-(12)-[NSImageView:0x7fd7bfa1eaf0]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031c49b0 H:[NSImageView:0x7fd7bfa1e850]-(12)-[NSImageView:0x7fd7bfa1eaf0]   (active)>",
    "<NSLayoutConstraint:0x6000031c47d0 H:[NSImageView:0x7fd7bef2a320]-(3)-[NSTextField:0x7fd7bfa1ed90]   (active)>",
    "<NSLayoutConstraint:0x6000031c4870 H:[NSTextField:0x7fd7bfa1ed90]-(10)-[NSImageView:0x7fd7bfa1e850]   (active)>",
    "<NSLayoutConstraint:0x6000031c4690 NSImageView:0x7fd7bef2a320.leading == HeliPort.WifiMenuItemView:0x7fd7bef29e00.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031c4a00 NSImageView:0x7fd7bfa1eaf0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef29e00.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c4f50 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef29e00.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031c49b0 H:[NSImageView:0x7fd7bfa1e850]-(12)-[NSImageView:0x7fd7bfa1eaf0]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031c4b40 H:|-(5)-[NSVisualEffectView:0x7fd7bef2a080](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bef29e00 )>",
    "<NSLayoutConstraint:0x6000031c4b90 NSVisualEffectView:0x7fd7bef2a080.right == HeliPort.WifiMenuItemView:0x7fd7bef29e00.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c4f50 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef29e00.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031c4b90 NSVisualEffectView:0x7fd7bef2a080.right == HeliPort.WifiMenuItemView:0x7fd7bef29e00.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031c4b40 H:|-(5)-[NSVisualEffectView:0x7fd7bef2a080](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bef29e00 )>",
    "<NSLayoutConstraint:0x6000031c4b90 NSVisualEffectView:0x7fd7bef2a080.right == HeliPort.WifiMenuItemView:0x7fd7bef29e00.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c4f50 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef29e00.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031c4b90 NSVisualEffectView:0x7fd7bef2a080.right == HeliPort.WifiMenuItemView:0x7fd7bef29e00.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031da030 NSImageView:0x7fd7bef29300.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031da300 NSImageView:0x7fd7bef2e520.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031da4e0 NSImageView:0x7fd7bef2e7c0.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031da3f0 H:[NSImageView:0x7fd7bef2e520]-(12)-[NSImageView:0x7fd7bef2e7c0]   (active)>",
    "<NSLayoutConstraint:0x6000031da210 H:[NSImageView:0x7fd7bef29300]-(3)-[NSTextField:0x7fd7bef2ea60]   (active)>",
    "<NSLayoutConstraint:0x6000031da2b0 H:[NSTextField:0x7fd7bef2ea60]-(10)-[NSImageView:0x7fd7bef2e520]   (active)>",
    "<NSLayoutConstraint:0x6000031da0d0 NSImageView:0x7fd7bef29300.leading == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031da440 NSImageView:0x7fd7bef2e7c0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031da990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031da4e0 NSImageView:0x7fd7bef2e7c0.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031da030 NSImageView:0x7fd7bef29300.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031da300 NSImageView:0x7fd7bef2e520.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031da4e0 NSImageView:0x7fd7bef2e7c0.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031da3f0 H:[NSImageView:0x7fd7bef2e520]-(12)-[NSImageView:0x7fd7bef2e7c0]   (active)>",
    "<NSLayoutConstraint:0x6000031da210 H:[NSImageView:0x7fd7bef29300]-(3)-[NSTextField:0x7fd7bef2ea60]   (active)>",
    "<NSLayoutConstraint:0x6000031da2b0 H:[NSTextField:0x7fd7bef2ea60]-(10)-[NSImageView:0x7fd7bef2e520]   (active)>",
    "<NSLayoutConstraint:0x6000031da0d0 NSImageView:0x7fd7bef29300.leading == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031da440 NSImageView:0x7fd7bef2e7c0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031da990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031da4e0 NSImageView:0x7fd7bef2e7c0.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031da030 NSImageView:0x7fd7bef29300.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031da300 NSImageView:0x7fd7bef2e520.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031da3f0 H:[NSImageView:0x7fd7bef2e520]-(12)-[NSImageView:0x7fd7bef2e7c0]   (active)>",
    "<NSLayoutConstraint:0x6000031da210 H:[NSImageView:0x7fd7bef29300]-(3)-[NSTextField:0x7fd7bef2ea60]   (active)>",
    "<NSLayoutConstraint:0x6000031da2b0 H:[NSTextField:0x7fd7bef2ea60]-(10)-[NSImageView:0x7fd7bef2e520]   (active)>",
    "<NSLayoutConstraint:0x6000031da0d0 NSImageView:0x7fd7bef29300.leading == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031da440 NSImageView:0x7fd7bef2e7c0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031da990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031da300 NSImageView:0x7fd7bef2e520.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031da030 NSImageView:0x7fd7bef29300.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031da300 NSImageView:0x7fd7bef2e520.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031da3f0 H:[NSImageView:0x7fd7bef2e520]-(12)-[NSImageView:0x7fd7bef2e7c0]   (active)>",
    "<NSLayoutConstraint:0x6000031da210 H:[NSImageView:0x7fd7bef29300]-(3)-[NSTextField:0x7fd7bef2ea60]   (active)>",
    "<NSLayoutConstraint:0x6000031da2b0 H:[NSTextField:0x7fd7bef2ea60]-(10)-[NSImageView:0x7fd7bef2e520]   (active)>",
    "<NSLayoutConstraint:0x6000031da0d0 NSImageView:0x7fd7bef29300.leading == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031da440 NSImageView:0x7fd7bef2e7c0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031da990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031da300 NSImageView:0x7fd7bef2e520.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031da030 NSImageView:0x7fd7bef29300.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031da3f0 H:[NSImageView:0x7fd7bef2e520]-(12)-[NSImageView:0x7fd7bef2e7c0]   (active)>",
    "<NSLayoutConstraint:0x6000031da210 H:[NSImageView:0x7fd7bef29300]-(3)-[NSTextField:0x7fd7bef2ea60]   (active)>",
    "<NSLayoutConstraint:0x6000031da2b0 H:[NSTextField:0x7fd7bef2ea60]-(10)-[NSImageView:0x7fd7bef2e520]   (active)>",
    "<NSLayoutConstraint:0x6000031da0d0 NSImageView:0x7fd7bef29300.leading == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031da440 NSImageView:0x7fd7bef2e7c0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031da990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031da030 NSImageView:0x7fd7bef29300.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031da030 NSImageView:0x7fd7bef29300.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031da3f0 H:[NSImageView:0x7fd7bef2e520]-(12)-[NSImageView:0x7fd7bef2e7c0]   (active)>",
    "<NSLayoutConstraint:0x6000031da210 H:[NSImageView:0x7fd7bef29300]-(3)-[NSTextField:0x7fd7bef2ea60]   (active)>",
    "<NSLayoutConstraint:0x6000031da2b0 H:[NSTextField:0x7fd7bef2ea60]-(10)-[NSImageView:0x7fd7bef2e520]   (active)>",
    "<NSLayoutConstraint:0x6000031da0d0 NSImageView:0x7fd7bef29300.leading == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031da440 NSImageView:0x7fd7bef2e7c0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031da990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031da030 NSImageView:0x7fd7bef29300.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031da3f0 H:[NSImageView:0x7fd7bef2e520]-(12)-[NSImageView:0x7fd7bef2e7c0]   (active)>",
    "<NSLayoutConstraint:0x6000031da210 H:[NSImageView:0x7fd7bef29300]-(3)-[NSTextField:0x7fd7bef2ea60]   (active)>",
    "<NSLayoutConstraint:0x6000031da2b0 H:[NSTextField:0x7fd7bef2ea60]-(10)-[NSImageView:0x7fd7bef2e520]   (active)>",
    "<NSLayoutConstraint:0x6000031da0d0 NSImageView:0x7fd7bef29300.leading == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031da440 NSImageView:0x7fd7bef2e7c0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031da990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031da3f0 H:[NSImageView:0x7fd7bef2e520]-(12)-[NSImageView:0x7fd7bef2e7c0]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031da3f0 H:[NSImageView:0x7fd7bef2e520]-(12)-[NSImageView:0x7fd7bef2e7c0]   (active)>",
    "<NSLayoutConstraint:0x6000031da210 H:[NSImageView:0x7fd7bef29300]-(3)-[NSTextField:0x7fd7bef2ea60]   (active)>",
    "<NSLayoutConstraint:0x6000031da2b0 H:[NSTextField:0x7fd7bef2ea60]-(10)-[NSImageView:0x7fd7bef2e520]   (active)>",
    "<NSLayoutConstraint:0x6000031da0d0 NSImageView:0x7fd7bef29300.leading == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031da440 NSImageView:0x7fd7bef2e7c0.trailing == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031da990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031da3f0 H:[NSImageView:0x7fd7bef2e520]-(12)-[NSImageView:0x7fd7bef2e7c0]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031da580 H:|-(5)-[NSVisualEffectView:0x7fd7bef29060](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bef1f5c0 )>",
    "<NSLayoutConstraint:0x6000031da5d0 NSVisualEffectView:0x7fd7bef29060.right == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031da990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031da5d0 NSVisualEffectView:0x7fd7bef29060.right == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031da580 H:|-(5)-[NSVisualEffectView:0x7fd7bef29060](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bef1f5c0 )>",
    "<NSLayoutConstraint:0x6000031da5d0 NSVisualEffectView:0x7fd7bef29060.right == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031da990 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031da5d0 NSVisualEffectView:0x7fd7bef29060.right == HeliPort.WifiMenuItemView:0x7fd7bef1f5c0.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b6cb0 NSImageView:0x7fd7bfb44ca0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b6fd0 NSImageView:0x7fd7bfb45380.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b7610 NSImageView:0x7fd7bfb45620.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031b7520 H:[NSImageView:0x7fd7bfb45380]-(12)-[NSImageView:0x7fd7bfb45620]   (active)>",
    "<NSLayoutConstraint:0x6000031b6da0 H:[NSImageView:0x7fd7bfb44ca0]-(3)-[NSTextField:0x7fd7bfb458c0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6e90 H:[NSTextField:0x7fd7bfb458c0]-(10)-[NSImageView:0x7fd7bfb45380]   (active)>",
    "<NSLayoutConstraint:0x6000031b6c60 NSImageView:0x7fd7bfb44ca0.leading == HeliPort.WifiMenuItemView:0x7fd7bfb44780.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7570 NSImageView:0x7fd7bfb45620.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb44780.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b0cd0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb44780.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b7610 NSImageView:0x7fd7bfb45620.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b6cb0 NSImageView:0x7fd7bfb44ca0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b6fd0 NSImageView:0x7fd7bfb45380.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b7610 NSImageView:0x7fd7bfb45620.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031b7520 H:[NSImageView:0x7fd7bfb45380]-(12)-[NSImageView:0x7fd7bfb45620]   (active)>",
    "<NSLayoutConstraint:0x6000031b6da0 H:[NSImageView:0x7fd7bfb44ca0]-(3)-[NSTextField:0x7fd7bfb458c0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6e90 H:[NSTextField:0x7fd7bfb458c0]-(10)-[NSImageView:0x7fd7bfb45380]   (active)>",
    "<NSLayoutConstraint:0x6000031b6c60 NSImageView:0x7fd7bfb44ca0.leading == HeliPort.WifiMenuItemView:0x7fd7bfb44780.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7570 NSImageView:0x7fd7bfb45620.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb44780.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b0cd0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb44780.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b7610 NSImageView:0x7fd7bfb45620.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b6cb0 NSImageView:0x7fd7bfb44ca0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b6fd0 NSImageView:0x7fd7bfb45380.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b7520 H:[NSImageView:0x7fd7bfb45380]-(12)-[NSImageView:0x7fd7bfb45620]   (active)>",
    "<NSLayoutConstraint:0x6000031b6da0 H:[NSImageView:0x7fd7bfb44ca0]-(3)-[NSTextField:0x7fd7bfb458c0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6e90 H:[NSTextField:0x7fd7bfb458c0]-(10)-[NSImageView:0x7fd7bfb45380]   (active)>",
    "<NSLayoutConstraint:0x6000031b6c60 NSImageView:0x7fd7bfb44ca0.leading == HeliPort.WifiMenuItemView:0x7fd7bfb44780.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7570 NSImageView:0x7fd7bfb45620.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb44780.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b0cd0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb44780.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b6fd0 NSImageView:0x7fd7bfb45380.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b6cb0 NSImageView:0x7fd7bfb44ca0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b6fd0 NSImageView:0x7fd7bfb45380.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b7520 H:[NSImageView:0x7fd7bfb45380]-(12)-[NSImageView:0x7fd7bfb45620]   (active)>",
    "<NSLayoutConstraint:0x6000031b6da0 H:[NSImageView:0x7fd7bfb44ca0]-(3)-[NSTextField:0x7fd7bfb458c0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6e90 H:[NSTextField:0x7fd7bfb458c0]-(10)-[NSImageView:0x7fd7bfb45380]   (active)>",
    "<NSLayoutConstraint:0x6000031b6c60 NSImageView:0x7fd7bfb44ca0.leading == HeliPort.WifiMenuItemView:0x7fd7bfb44780.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7570 NSImageView:0x7fd7bfb45620.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb44780.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b0cd0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb44780.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b6fd0 NSImageView:0x7fd7bfb45380.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b6cb0 NSImageView:0x7fd7bfb44ca0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b7520 H:[NSImageView:0x7fd7bfb45380]-(12)-[NSImageView:0x7fd7bfb45620]   (active)>",
    "<NSLayoutConstraint:0x6000031b6da0 H:[NSImageView:0x7fd7bfb44ca0]-(3)-[NSTextField:0x7fd7bfb458c0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6e90 H:[NSTextField:0x7fd7bfb458c0]-(10)-[NSImageView:0x7fd7bfb45380]   (active)>",
    "<NSLayoutConstraint:0x6000031b6c60 NSImageView:0x7fd7bfb44ca0.leading == HeliPort.WifiMenuItemView:0x7fd7bfb44780.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7570 NSImageView:0x7fd7bfb45620.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb44780.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b0cd0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb44780.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b6cb0 NSImageView:0x7fd7bfb44ca0.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b6cb0 NSImageView:0x7fd7bfb44ca0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b7520 H:[NSImageView:0x7fd7bfb45380]-(12)-[NSImageView:0x7fd7bfb45620]   (active)>",
    "<NSLayoutConstraint:0x6000031b6da0 H:[NSImageView:0x7fd7bfb44ca0]-(3)-[NSTextField:0x7fd7bfb458c0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6e90 H:[NSTextField:0x7fd7bfb458c0]-(10)-[NSImageView:0x7fd7bfb45380]   (active)>",
    "<NSLayoutConstraint:0x6000031b6c60 NSImageView:0x7fd7bfb44ca0.leading == HeliPort.WifiMenuItemView:0x7fd7bfb44780.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7570 NSImageView:0x7fd7bfb45620.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb44780.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b0cd0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb44780.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b6cb0 NSImageView:0x7fd7bfb44ca0.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b7520 H:[NSImageView:0x7fd7bfb45380]-(12)-[NSImageView:0x7fd7bfb45620]   (active)>",
    "<NSLayoutConstraint:0x6000031b6da0 H:[NSImageView:0x7fd7bfb44ca0]-(3)-[NSTextField:0x7fd7bfb458c0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6e90 H:[NSTextField:0x7fd7bfb458c0]-(10)-[NSImageView:0x7fd7bfb45380]   (active)>",
    "<NSLayoutConstraint:0x6000031b6c60 NSImageView:0x7fd7bfb44ca0.leading == HeliPort.WifiMenuItemView:0x7fd7bfb44780.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7570 NSImageView:0x7fd7bfb45620.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb44780.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b0cd0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb44780.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b7520 H:[NSImageView:0x7fd7bfb45380]-(12)-[NSImageView:0x7fd7bfb45620]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b7520 H:[NSImageView:0x7fd7bfb45380]-(12)-[NSImageView:0x7fd7bfb45620]   (active)>",
    "<NSLayoutConstraint:0x6000031b6da0 H:[NSImageView:0x7fd7bfb44ca0]-(3)-[NSTextField:0x7fd7bfb458c0]   (active)>",
    "<NSLayoutConstraint:0x6000031b6e90 H:[NSTextField:0x7fd7bfb458c0]-(10)-[NSImageView:0x7fd7bfb45380]   (active)>",
    "<NSLayoutConstraint:0x6000031b6c60 NSImageView:0x7fd7bfb44ca0.leading == HeliPort.WifiMenuItemView:0x7fd7bfb44780.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b7570 NSImageView:0x7fd7bfb45620.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb44780.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b0cd0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb44780.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b7520 H:[NSImageView:0x7fd7bfb45380]-(12)-[NSImageView:0x7fd7bfb45620]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b76b0 H:|-(5)-[NSVisualEffectView:0x7fd7bfb44a00](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfb44780 )>",
    "<NSLayoutConstraint:0x6000031b77a0 NSVisualEffectView:0x7fd7bfb44a00.right == HeliPort.WifiMenuItemView:0x7fd7bfb44780.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b0cd0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb44780.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b77a0 NSVisualEffectView:0x7fd7bfb44a00.right == HeliPort.WifiMenuItemView:0x7fd7bfb44780.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b76b0 H:|-(5)-[NSVisualEffectView:0x7fd7bfb44a00](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfb44780 )>",
    "<NSLayoutConstraint:0x6000031b77a0 NSVisualEffectView:0x7fd7bfb44a00.right == HeliPort.WifiMenuItemView:0x7fd7bfb44780.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031b0cd0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb44780.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b77a0 NSVisualEffectView:0x7fd7bfb44a00.right == HeliPort.WifiMenuItemView:0x7fd7bfb44780.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b1400 NSImageView:0x7fd7bfb46710.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bcfa0 NSImageView:0x7fd7bfb469b0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bc5f0 NSImageView:0x7fd7bfb46c50.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031bc640 H:[NSImageView:0x7fd7bfb469b0]-(12)-[NSImageView:0x7fd7bfb46c50]   (active)>",
    "<NSLayoutConstraint:0x6000031b14a0 H:[NSImageView:0x7fd7bfb46710]-(3)-[NSTextField:0x7fd7bfb48b30]   (active)>",
    "<NSLayoutConstraint:0x6000031b0050 H:[NSTextField:0x7fd7bfb48b30]-(10)-[NSImageView:0x7fd7bfb469b0]   (active)>",
    "<NSLayoutConstraint:0x6000031b0640 NSImageView:0x7fd7bfb46710.leading == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bcd70 NSImageView:0x7fd7bfb46c50.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bd180 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb485f0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bc5f0 NSImageView:0x7fd7bfb46c50.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b1400 NSImageView:0x7fd7bfb46710.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bcfa0 NSImageView:0x7fd7bfb469b0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bc5f0 NSImageView:0x7fd7bfb46c50.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031bc640 H:[NSImageView:0x7fd7bfb469b0]-(12)-[NSImageView:0x7fd7bfb46c50]   (active)>",
    "<NSLayoutConstraint:0x6000031b14a0 H:[NSImageView:0x7fd7bfb46710]-(3)-[NSTextField:0x7fd7bfb48b30]   (active)>",
    "<NSLayoutConstraint:0x6000031b0050 H:[NSTextField:0x7fd7bfb48b30]-(10)-[NSImageView:0x7fd7bfb469b0]   (active)>",
    "<NSLayoutConstraint:0x6000031b0640 NSImageView:0x7fd7bfb46710.leading == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bcd70 NSImageView:0x7fd7bfb46c50.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bd180 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb485f0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bc5f0 NSImageView:0x7fd7bfb46c50.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b1400 NSImageView:0x7fd7bfb46710.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bcfa0 NSImageView:0x7fd7bfb469b0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bc640 H:[NSImageView:0x7fd7bfb469b0]-(12)-[NSImageView:0x7fd7bfb46c50]   (active)>",
    "<NSLayoutConstraint:0x6000031b14a0 H:[NSImageView:0x7fd7bfb46710]-(3)-[NSTextField:0x7fd7bfb48b30]   (active)>",
    "<NSLayoutConstraint:0x6000031b0050 H:[NSTextField:0x7fd7bfb48b30]-(10)-[NSImageView:0x7fd7bfb469b0]   (active)>",
    "<NSLayoutConstraint:0x6000031b0640 NSImageView:0x7fd7bfb46710.leading == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bcd70 NSImageView:0x7fd7bfb46c50.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bd180 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb485f0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bcfa0 NSImageView:0x7fd7bfb469b0.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b1400 NSImageView:0x7fd7bfb46710.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bcfa0 NSImageView:0x7fd7bfb469b0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bc640 H:[NSImageView:0x7fd7bfb469b0]-(12)-[NSImageView:0x7fd7bfb46c50]   (active)>",
    "<NSLayoutConstraint:0x6000031b14a0 H:[NSImageView:0x7fd7bfb46710]-(3)-[NSTextField:0x7fd7bfb48b30]   (active)>",
    "<NSLayoutConstraint:0x6000031b0050 H:[NSTextField:0x7fd7bfb48b30]-(10)-[NSImageView:0x7fd7bfb469b0]   (active)>",
    "<NSLayoutConstraint:0x6000031b0640 NSImageView:0x7fd7bfb46710.leading == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bcd70 NSImageView:0x7fd7bfb46c50.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bd180 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb485f0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bcfa0 NSImageView:0x7fd7bfb469b0.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b1400 NSImageView:0x7fd7bfb46710.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bc640 H:[NSImageView:0x7fd7bfb469b0]-(12)-[NSImageView:0x7fd7bfb46c50]   (active)>",
    "<NSLayoutConstraint:0x6000031b14a0 H:[NSImageView:0x7fd7bfb46710]-(3)-[NSTextField:0x7fd7bfb48b30]   (active)>",
    "<NSLayoutConstraint:0x6000031b0050 H:[NSTextField:0x7fd7bfb48b30]-(10)-[NSImageView:0x7fd7bfb469b0]   (active)>",
    "<NSLayoutConstraint:0x6000031b0640 NSImageView:0x7fd7bfb46710.leading == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bcd70 NSImageView:0x7fd7bfb46c50.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bd180 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb485f0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b1400 NSImageView:0x7fd7bfb46710.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b1400 NSImageView:0x7fd7bfb46710.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bc640 H:[NSImageView:0x7fd7bfb469b0]-(12)-[NSImageView:0x7fd7bfb46c50]   (active)>",
    "<NSLayoutConstraint:0x6000031b14a0 H:[NSImageView:0x7fd7bfb46710]-(3)-[NSTextField:0x7fd7bfb48b30]   (active)>",
    "<NSLayoutConstraint:0x6000031b0050 H:[NSTextField:0x7fd7bfb48b30]-(10)-[NSImageView:0x7fd7bfb469b0]   (active)>",
    "<NSLayoutConstraint:0x6000031b0640 NSImageView:0x7fd7bfb46710.leading == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bcd70 NSImageView:0x7fd7bfb46c50.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bd180 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb485f0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b1400 NSImageView:0x7fd7bfb46710.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bc640 H:[NSImageView:0x7fd7bfb469b0]-(12)-[NSImageView:0x7fd7bfb46c50]   (active)>",
    "<NSLayoutConstraint:0x6000031b14a0 H:[NSImageView:0x7fd7bfb46710]-(3)-[NSTextField:0x7fd7bfb48b30]   (active)>",
    "<NSLayoutConstraint:0x6000031b0050 H:[NSTextField:0x7fd7bfb48b30]-(10)-[NSImageView:0x7fd7bfb469b0]   (active)>",
    "<NSLayoutConstraint:0x6000031b0640 NSImageView:0x7fd7bfb46710.leading == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bcd70 NSImageView:0x7fd7bfb46c50.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bd180 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb485f0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bc640 H:[NSImageView:0x7fd7bfb469b0]-(12)-[NSImageView:0x7fd7bfb46c50]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bc640 H:[NSImageView:0x7fd7bfb469b0]-(12)-[NSImageView:0x7fd7bfb46c50]   (active)>",
    "<NSLayoutConstraint:0x6000031b14a0 H:[NSImageView:0x7fd7bfb46710]-(3)-[NSTextField:0x7fd7bfb48b30]   (active)>",
    "<NSLayoutConstraint:0x6000031b0050 H:[NSTextField:0x7fd7bfb48b30]-(10)-[NSImageView:0x7fd7bfb469b0]   (active)>",
    "<NSLayoutConstraint:0x6000031b0640 NSImageView:0x7fd7bfb46710.leading == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bcd70 NSImageView:0x7fd7bfb46c50.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bd180 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb485f0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bc640 H:[NSImageView:0x7fd7bfb469b0]-(12)-[NSImageView:0x7fd7bfb46c50]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bcaf0 H:|-(5)-[NSVisualEffectView:0x7fd7bfb473b0](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfb485f0 )>",
    "<NSLayoutConstraint:0x6000031bcc80 NSVisualEffectView:0x7fd7bfb473b0.right == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bd180 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb485f0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bcc80 NSVisualEffectView:0x7fd7bfb473b0.right == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bcaf0 H:|-(5)-[NSVisualEffectView:0x7fd7bfb473b0](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfb485f0 )>",
    "<NSLayoutConstraint:0x6000031bcc80 NSVisualEffectView:0x7fd7bfb473b0.right == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bd180 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb485f0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bcc80 NSVisualEffectView:0x7fd7bfb473b0.right == HeliPort.WifiMenuItemView:0x7fd7bfb485f0.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031a8550 NSImageView:0x7fd7bfc06040.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031a8730 NSImageView:0x7fd7bfc062e0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031a8910 NSImageView:0x7fd7bfc06580.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031a8820 H:[NSImageView:0x7fd7bfc062e0]-(12)-[NSImageView:0x7fd7bfc06580]   (active)>",
    "<NSLayoutConstraint:0x6000031a8640 H:[NSImageView:0x7fd7bfc06040]-(3)-[NSTextField:0x7fd7bfc06820]   (active)>",
    "<NSLayoutConstraint:0x6000031a86e0 H:[NSTextField:0x7fd7bfc06820]-(10)-[NSImageView:0x7fd7bfc062e0]   (active)>",
    "<NSLayoutConstraint:0x6000031a8500 NSImageView:0x7fd7bfc06040.leading == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031a8870 NSImageView:0x7fd7bfc06580.trailing == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c2120 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfc05b20.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031a8910 NSImageView:0x7fd7bfc06580.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031a8550 NSImageView:0x7fd7bfc06040.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031a8730 NSImageView:0x7fd7bfc062e0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031a8910 NSImageView:0x7fd7bfc06580.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031a8820 H:[NSImageView:0x7fd7bfc062e0]-(12)-[NSImageView:0x7fd7bfc06580]   (active)>",
    "<NSLayoutConstraint:0x6000031a8640 H:[NSImageView:0x7fd7bfc06040]-(3)-[NSTextField:0x7fd7bfc06820]   (active)>",
    "<NSLayoutConstraint:0x6000031a86e0 H:[NSTextField:0x7fd7bfc06820]-(10)-[NSImageView:0x7fd7bfc062e0]   (active)>",
    "<NSLayoutConstraint:0x6000031a8500 NSImageView:0x7fd7bfc06040.leading == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031a8870 NSImageView:0x7fd7bfc06580.trailing == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c2120 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfc05b20.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031a8910 NSImageView:0x7fd7bfc06580.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031a8550 NSImageView:0x7fd7bfc06040.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031a8730 NSImageView:0x7fd7bfc062e0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031a8820 H:[NSImageView:0x7fd7bfc062e0]-(12)-[NSImageView:0x7fd7bfc06580]   (active)>",
    "<NSLayoutConstraint:0x6000031a8640 H:[NSImageView:0x7fd7bfc06040]-(3)-[NSTextField:0x7fd7bfc06820]   (active)>",
    "<NSLayoutConstraint:0x6000031a86e0 H:[NSTextField:0x7fd7bfc06820]-(10)-[NSImageView:0x7fd7bfc062e0]   (active)>",
    "<NSLayoutConstraint:0x6000031a8500 NSImageView:0x7fd7bfc06040.leading == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031a8870 NSImageView:0x7fd7bfc06580.trailing == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c2120 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfc05b20.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031a8730 NSImageView:0x7fd7bfc062e0.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031a8550 NSImageView:0x7fd7bfc06040.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031a8730 NSImageView:0x7fd7bfc062e0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031a8820 H:[NSImageView:0x7fd7bfc062e0]-(12)-[NSImageView:0x7fd7bfc06580]   (active)>",
    "<NSLayoutConstraint:0x6000031a8640 H:[NSImageView:0x7fd7bfc06040]-(3)-[NSTextField:0x7fd7bfc06820]   (active)>",
    "<NSLayoutConstraint:0x6000031a86e0 H:[NSTextField:0x7fd7bfc06820]-(10)-[NSImageView:0x7fd7bfc062e0]   (active)>",
    "<NSLayoutConstraint:0x6000031a8500 NSImageView:0x7fd7bfc06040.leading == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031a8870 NSImageView:0x7fd7bfc06580.trailing == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c2120 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfc05b20.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031a8730 NSImageView:0x7fd7bfc062e0.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031a8550 NSImageView:0x7fd7bfc06040.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031a8820 H:[NSImageView:0x7fd7bfc062e0]-(12)-[NSImageView:0x7fd7bfc06580]   (active)>",
    "<NSLayoutConstraint:0x6000031a8640 H:[NSImageView:0x7fd7bfc06040]-(3)-[NSTextField:0x7fd7bfc06820]   (active)>",
    "<NSLayoutConstraint:0x6000031a86e0 H:[NSTextField:0x7fd7bfc06820]-(10)-[NSImageView:0x7fd7bfc062e0]   (active)>",
    "<NSLayoutConstraint:0x6000031a8500 NSImageView:0x7fd7bfc06040.leading == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031a8870 NSImageView:0x7fd7bfc06580.trailing == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c2120 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfc05b20.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031a8550 NSImageView:0x7fd7bfc06040.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031a8550 NSImageView:0x7fd7bfc06040.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031a8820 H:[NSImageView:0x7fd7bfc062e0]-(12)-[NSImageView:0x7fd7bfc06580]   (active)>",
    "<NSLayoutConstraint:0x6000031a8640 H:[NSImageView:0x7fd7bfc06040]-(3)-[NSTextField:0x7fd7bfc06820]   (active)>",
    "<NSLayoutConstraint:0x6000031a86e0 H:[NSTextField:0x7fd7bfc06820]-(10)-[NSImageView:0x7fd7bfc062e0]   (active)>",
    "<NSLayoutConstraint:0x6000031a8500 NSImageView:0x7fd7bfc06040.leading == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031a8870 NSImageView:0x7fd7bfc06580.trailing == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c2120 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfc05b20.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031a8550 NSImageView:0x7fd7bfc06040.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031a8820 H:[NSImageView:0x7fd7bfc062e0]-(12)-[NSImageView:0x7fd7bfc06580]   (active)>",
    "<NSLayoutConstraint:0x6000031a8640 H:[NSImageView:0x7fd7bfc06040]-(3)-[NSTextField:0x7fd7bfc06820]   (active)>",
    "<NSLayoutConstraint:0x6000031a86e0 H:[NSTextField:0x7fd7bfc06820]-(10)-[NSImageView:0x7fd7bfc062e0]   (active)>",
    "<NSLayoutConstraint:0x6000031a8500 NSImageView:0x7fd7bfc06040.leading == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031a8870 NSImageView:0x7fd7bfc06580.trailing == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c2120 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfc05b20.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031a8820 H:[NSImageView:0x7fd7bfc062e0]-(12)-[NSImageView:0x7fd7bfc06580]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031a8820 H:[NSImageView:0x7fd7bfc062e0]-(12)-[NSImageView:0x7fd7bfc06580]   (active)>",
    "<NSLayoutConstraint:0x6000031a8640 H:[NSImageView:0x7fd7bfc06040]-(3)-[NSTextField:0x7fd7bfc06820]   (active)>",
    "<NSLayoutConstraint:0x6000031a86e0 H:[NSTextField:0x7fd7bfc06820]-(10)-[NSImageView:0x7fd7bfc062e0]   (active)>",
    "<NSLayoutConstraint:0x6000031a8500 NSImageView:0x7fd7bfc06040.leading == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031a8870 NSImageView:0x7fd7bfc06580.trailing == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c2120 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfc05b20.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031a8820 H:[NSImageView:0x7fd7bfc062e0]-(12)-[NSImageView:0x7fd7bfc06580]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031a89b0 H:|-(5)-[NSVisualEffectView:0x7fd7bfc05da0](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfc05b20 )>",
    "<NSLayoutConstraint:0x6000031a8a00 NSVisualEffectView:0x7fd7bfc05da0.right == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c2120 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfc05b20.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031a8a00 NSVisualEffectView:0x7fd7bfc05da0.right == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031a89b0 H:|-(5)-[NSVisualEffectView:0x7fd7bfc05da0](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfc05b20 )>",
    "<NSLayoutConstraint:0x6000031a8a00 NSVisualEffectView:0x7fd7bfc05da0.right == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031c2120 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfc05b20.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031a8a00 NSVisualEffectView:0x7fd7bfc05da0.right == HeliPort.WifiMenuItemView:0x7fd7bfc05b20.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031cef80 NSImageView:0x7fd7bfa2c430.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031cfbb0 NSImageView:0x7fd7bfa2c6d0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bbac0 NSImageView:0x7fd7bfa2c970.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031b8280 H:[NSImageView:0x7fd7bfa2c6d0]-(12)-[NSImageView:0x7fd7bfa2c970]   (active)>",
    "<NSLayoutConstraint:0x6000031cf660 H:[NSImageView:0x7fd7bfa2c430]-(3)-[NSTextField:0x7fd7bfa2b1b0]   (active)>",
    "<NSLayoutConstraint:0x6000031cfb60 H:[NSTextField:0x7fd7bfa2b1b0]-(10)-[NSImageView:0x7fd7bfa2c6d0]   (active)>",
    "<NSLayoutConstraint:0x6000031cef30 NSImageView:0x7fd7bfa2c430.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b8230 NSImageView:0x7fd7bfa2c970.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031babc0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bbac0 NSImageView:0x7fd7bfa2c970.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031cef80 NSImageView:0x7fd7bfa2c430.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031cfbb0 NSImageView:0x7fd7bfa2c6d0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bbac0 NSImageView:0x7fd7bfa2c970.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031b8280 H:[NSImageView:0x7fd7bfa2c6d0]-(12)-[NSImageView:0x7fd7bfa2c970]   (active)>",
    "<NSLayoutConstraint:0x6000031cf660 H:[NSImageView:0x7fd7bfa2c430]-(3)-[NSTextField:0x7fd7bfa2b1b0]   (active)>",
    "<NSLayoutConstraint:0x6000031cfb60 H:[NSTextField:0x7fd7bfa2b1b0]-(10)-[NSImageView:0x7fd7bfa2c6d0]   (active)>",
    "<NSLayoutConstraint:0x6000031cef30 NSImageView:0x7fd7bfa2c430.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b8230 NSImageView:0x7fd7bfa2c970.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031babc0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bbac0 NSImageView:0x7fd7bfa2c970.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031cef80 NSImageView:0x7fd7bfa2c430.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031cfbb0 NSImageView:0x7fd7bfa2c6d0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b8280 H:[NSImageView:0x7fd7bfa2c6d0]-(12)-[NSImageView:0x7fd7bfa2c970]   (active)>",
    "<NSLayoutConstraint:0x6000031cf660 H:[NSImageView:0x7fd7bfa2c430]-(3)-[NSTextField:0x7fd7bfa2b1b0]   (active)>",
    "<NSLayoutConstraint:0x6000031cfb60 H:[NSTextField:0x7fd7bfa2b1b0]-(10)-[NSImageView:0x7fd7bfa2c6d0]   (active)>",
    "<NSLayoutConstraint:0x6000031cef30 NSImageView:0x7fd7bfa2c430.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b8230 NSImageView:0x7fd7bfa2c970.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031babc0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031cfbb0 NSImageView:0x7fd7bfa2c6d0.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031cef80 NSImageView:0x7fd7bfa2c430.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031cfbb0 NSImageView:0x7fd7bfa2c6d0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b8280 H:[NSImageView:0x7fd7bfa2c6d0]-(12)-[NSImageView:0x7fd7bfa2c970]   (active)>",
    "<NSLayoutConstraint:0x6000031cf660 H:[NSImageView:0x7fd7bfa2c430]-(3)-[NSTextField:0x7fd7bfa2b1b0]   (active)>",
    "<NSLayoutConstraint:0x6000031cfb60 H:[NSTextField:0x7fd7bfa2b1b0]-(10)-[NSImageView:0x7fd7bfa2c6d0]   (active)>",
    "<NSLayoutConstraint:0x6000031cef30 NSImageView:0x7fd7bfa2c430.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b8230 NSImageView:0x7fd7bfa2c970.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031babc0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031cfbb0 NSImageView:0x7fd7bfa2c6d0.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031cef80 NSImageView:0x7fd7bfa2c430.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b8280 H:[NSImageView:0x7fd7bfa2c6d0]-(12)-[NSImageView:0x7fd7bfa2c970]   (active)>",
    "<NSLayoutConstraint:0x6000031cf660 H:[NSImageView:0x7fd7bfa2c430]-(3)-[NSTextField:0x7fd7bfa2b1b0]   (active)>",
    "<NSLayoutConstraint:0x6000031cfb60 H:[NSTextField:0x7fd7bfa2b1b0]-(10)-[NSImageView:0x7fd7bfa2c6d0]   (active)>",
    "<NSLayoutConstraint:0x6000031cef30 NSImageView:0x7fd7bfa2c430.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b8230 NSImageView:0x7fd7bfa2c970.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031babc0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031cef80 NSImageView:0x7fd7bfa2c430.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031cef80 NSImageView:0x7fd7bfa2c430.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b8280 H:[NSImageView:0x7fd7bfa2c6d0]-(12)-[NSImageView:0x7fd7bfa2c970]   (active)>",
    "<NSLayoutConstraint:0x6000031cf660 H:[NSImageView:0x7fd7bfa2c430]-(3)-[NSTextField:0x7fd7bfa2b1b0]   (active)>",
    "<NSLayoutConstraint:0x6000031cfb60 H:[NSTextField:0x7fd7bfa2b1b0]-(10)-[NSImageView:0x7fd7bfa2c6d0]   (active)>",
    "<NSLayoutConstraint:0x6000031cef30 NSImageView:0x7fd7bfa2c430.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b8230 NSImageView:0x7fd7bfa2c970.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031babc0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031cef80 NSImageView:0x7fd7bfa2c430.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b8280 H:[NSImageView:0x7fd7bfa2c6d0]-(12)-[NSImageView:0x7fd7bfa2c970]   (active)>",
    "<NSLayoutConstraint:0x6000031cf660 H:[NSImageView:0x7fd7bfa2c430]-(3)-[NSTextField:0x7fd7bfa2b1b0]   (active)>",
    "<NSLayoutConstraint:0x6000031cfb60 H:[NSTextField:0x7fd7bfa2b1b0]-(10)-[NSImageView:0x7fd7bfa2c6d0]   (active)>",
    "<NSLayoutConstraint:0x6000031cef30 NSImageView:0x7fd7bfa2c430.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b8230 NSImageView:0x7fd7bfa2c970.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031babc0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b8280 H:[NSImageView:0x7fd7bfa2c6d0]-(12)-[NSImageView:0x7fd7bfa2c970]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b8280 H:[NSImageView:0x7fd7bfa2c6d0]-(12)-[NSImageView:0x7fd7bfa2c970]   (active)>",
    "<NSLayoutConstraint:0x6000031cf660 H:[NSImageView:0x7fd7bfa2c430]-(3)-[NSTextField:0x7fd7bfa2b1b0]   (active)>",
    "<NSLayoutConstraint:0x6000031cfb60 H:[NSTextField:0x7fd7bfa2b1b0]-(10)-[NSImageView:0x7fd7bfa2c6d0]   (active)>",
    "<NSLayoutConstraint:0x6000031cef30 NSImageView:0x7fd7bfa2c430.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031b8230 NSImageView:0x7fd7bfa2c970.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031babc0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b8280 H:[NSImageView:0x7fd7bfa2c6d0]-(12)-[NSImageView:0x7fd7bfa2c970]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bba20 H:|-(5)-[NSVisualEffectView:0x7fd7bfa2c190](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfa2bc20 )>",
    "<NSLayoutConstraint:0x6000031bb9d0 NSVisualEffectView:0x7fd7bfa2c190.right == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031babc0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bb9d0 NSVisualEffectView:0x7fd7bfa2c190.right == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bba20 H:|-(5)-[NSVisualEffectView:0x7fd7bfa2c190](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfa2bc20 )>",
    "<NSLayoutConstraint:0x6000031bb9d0 NSVisualEffectView:0x7fd7bfa2c190.right == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031babc0 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bb9d0 NSVisualEffectView:0x7fd7bfa2c190.right == HeliPort.WifiMenuItemView:0x7fd7bfa2bc20.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b9950 NSImageView:0x7fd7bfa2d7d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b9d10 NSImageView:0x7fd7bfa2da70.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bb7a0 NSImageView:0x7fd7bfa2f770.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031b98b0 H:[NSImageView:0x7fd7bfa2da70]-(12)-[NSImageView:0x7fd7bfa2f770]   (active)>",
    "<NSLayoutConstraint:0x6000031b9db0 H:[NSImageView:0x7fd7bfa2d7d0]-(3)-[NSTextField:0x7fd7bfa2fa10]   (active)>",
    "<NSLayoutConstraint:0x6000031b9d60 H:[NSTextField:0x7fd7bfa2fa10]-(10)-[NSImageView:0x7fd7bfa2da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b99a0 NSImageView:0x7fd7bfa2d7d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bb890 NSImageView:0x7fd7bfa2f770.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bbf70 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bb7a0 NSImageView:0x7fd7bfa2f770.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b9950 NSImageView:0x7fd7bfa2d7d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b9d10 NSImageView:0x7fd7bfa2da70.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bb7a0 NSImageView:0x7fd7bfa2f770.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031b98b0 H:[NSImageView:0x7fd7bfa2da70]-(12)-[NSImageView:0x7fd7bfa2f770]   (active)>",
    "<NSLayoutConstraint:0x6000031b9db0 H:[NSImageView:0x7fd7bfa2d7d0]-(3)-[NSTextField:0x7fd7bfa2fa10]   (active)>",
    "<NSLayoutConstraint:0x6000031b9d60 H:[NSTextField:0x7fd7bfa2fa10]-(10)-[NSImageView:0x7fd7bfa2da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b99a0 NSImageView:0x7fd7bfa2d7d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bb890 NSImageView:0x7fd7bfa2f770.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bbf70 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bb7a0 NSImageView:0x7fd7bfa2f770.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b9950 NSImageView:0x7fd7bfa2d7d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b9d10 NSImageView:0x7fd7bfa2da70.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b98b0 H:[NSImageView:0x7fd7bfa2da70]-(12)-[NSImageView:0x7fd7bfa2f770]   (active)>",
    "<NSLayoutConstraint:0x6000031b9db0 H:[NSImageView:0x7fd7bfa2d7d0]-(3)-[NSTextField:0x7fd7bfa2fa10]   (active)>",
    "<NSLayoutConstraint:0x6000031b9d60 H:[NSTextField:0x7fd7bfa2fa10]-(10)-[NSImageView:0x7fd7bfa2da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b99a0 NSImageView:0x7fd7bfa2d7d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bb890 NSImageView:0x7fd7bfa2f770.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bbf70 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b9d10 NSImageView:0x7fd7bfa2da70.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b9950 NSImageView:0x7fd7bfa2d7d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b9d10 NSImageView:0x7fd7bfa2da70.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031b98b0 H:[NSImageView:0x7fd7bfa2da70]-(12)-[NSImageView:0x7fd7bfa2f770]   (active)>",
    "<NSLayoutConstraint:0x6000031b9db0 H:[NSImageView:0x7fd7bfa2d7d0]-(3)-[NSTextField:0x7fd7bfa2fa10]   (active)>",
    "<NSLayoutConstraint:0x6000031b9d60 H:[NSTextField:0x7fd7bfa2fa10]-(10)-[NSImageView:0x7fd7bfa2da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b99a0 NSImageView:0x7fd7bfa2d7d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bb890 NSImageView:0x7fd7bfa2f770.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bbf70 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b9d10 NSImageView:0x7fd7bfa2da70.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b9950 NSImageView:0x7fd7bfa2d7d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b98b0 H:[NSImageView:0x7fd7bfa2da70]-(12)-[NSImageView:0x7fd7bfa2f770]   (active)>",
    "<NSLayoutConstraint:0x6000031b9db0 H:[NSImageView:0x7fd7bfa2d7d0]-(3)-[NSTextField:0x7fd7bfa2fa10]   (active)>",
    "<NSLayoutConstraint:0x6000031b9d60 H:[NSTextField:0x7fd7bfa2fa10]-(10)-[NSImageView:0x7fd7bfa2da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b99a0 NSImageView:0x7fd7bfa2d7d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bb890 NSImageView:0x7fd7bfa2f770.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bbf70 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b9950 NSImageView:0x7fd7bfa2d7d0.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b9950 NSImageView:0x7fd7bfa2d7d0.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031b98b0 H:[NSImageView:0x7fd7bfa2da70]-(12)-[NSImageView:0x7fd7bfa2f770]   (active)>",
    "<NSLayoutConstraint:0x6000031b9db0 H:[NSImageView:0x7fd7bfa2d7d0]-(3)-[NSTextField:0x7fd7bfa2fa10]   (active)>",
    "<NSLayoutConstraint:0x6000031b9d60 H:[NSTextField:0x7fd7bfa2fa10]-(10)-[NSImageView:0x7fd7bfa2da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b99a0 NSImageView:0x7fd7bfa2d7d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bb890 NSImageView:0x7fd7bfa2f770.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bbf70 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b9950 NSImageView:0x7fd7bfa2d7d0.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031b98b0 H:[NSImageView:0x7fd7bfa2da70]-(12)-[NSImageView:0x7fd7bfa2f770]   (active)>",
    "<NSLayoutConstraint:0x6000031b9db0 H:[NSImageView:0x7fd7bfa2d7d0]-(3)-[NSTextField:0x7fd7bfa2fa10]   (active)>",
    "<NSLayoutConstraint:0x6000031b9d60 H:[NSTextField:0x7fd7bfa2fa10]-(10)-[NSImageView:0x7fd7bfa2da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b99a0 NSImageView:0x7fd7bfa2d7d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bb890 NSImageView:0x7fd7bfa2f770.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bbf70 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031b98b0 H:[NSImageView:0x7fd7bfa2da70]-(12)-[NSImageView:0x7fd7bfa2f770]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031b98b0 H:[NSImageView:0x7fd7bfa2da70]-(12)-[NSImageView:0x7fd7bfa2f770]   (active)>",
    "<NSLayoutConstraint:0x6000031b9db0 H:[NSImageView:0x7fd7bfa2d7d0]-(3)-[NSTextField:0x7fd7bfa2fa10]   (active)>",
    "<NSLayoutConstraint:0x6000031b9d60 H:[NSTextField:0x7fd7bfa2fa10]-(10)-[NSImageView:0x7fd7bfa2da70]   (active)>",
    "<NSLayoutConstraint:0x6000031b99a0 NSImageView:0x7fd7bfa2d7d0.leading == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bb890 NSImageView:0x7fd7bfa2f770.trailing == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bbf70 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031b98b0 H:[NSImageView:0x7fd7bfa2da70]-(12)-[NSImageView:0x7fd7bfa2f770]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031ba530 H:|-(5)-[NSVisualEffectView:0x7fd7bfa2f230](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0 )>",
    "<NSLayoutConstraint:0x6000031ba580 NSVisualEffectView:0x7fd7bfa2f230.right == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bbf70 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031ba580 NSVisualEffectView:0x7fd7bfa2f230.right == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031ba530 H:|-(5)-[NSVisualEffectView:0x7fd7bfa2f230](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0 )>",
    "<NSLayoutConstraint:0x6000031ba580 NSVisualEffectView:0x7fd7bfa2f230.right == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bbf70 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031ba580 NSVisualEffectView:0x7fd7bfa2f230.right == HeliPort.WifiMenuItemView:0x7fd7bfa2e3c0.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bf1b0 NSImageView:0x7fd7bfb4b400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bf6b0 NSImageView:0x7fd7bfb4b6a0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bfb60 NSImageView:0x7fd7bfb4b940.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031bfa70 H:[NSImageView:0x7fd7bfb4b6a0]-(12)-[NSImageView:0x7fd7bfb4b940]   (active)>",
    "<NSLayoutConstraint:0x6000031bf2a0 H:[NSImageView:0x7fd7bfb4b400]-(3)-[NSTextField:0x7fd7bfb4bbe0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf4d0 H:[NSTextField:0x7fd7bfb4bbe0]-(10)-[NSImageView:0x7fd7bfb4b6a0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf160 NSImageView:0x7fd7bfb4b400.leading == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bfac0 NSImageView:0x7fd7bfb4b940.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bf070 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bfb60 NSImageView:0x7fd7bfb4b940.width == 18   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bf1b0 NSImageView:0x7fd7bfb4b400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bf6b0 NSImageView:0x7fd7bfb4b6a0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bfb60 NSImageView:0x7fd7bfb4b940.width == 18   (active)>",
    "<NSLayoutConstraint:0x6000031bfa70 H:[NSImageView:0x7fd7bfb4b6a0]-(12)-[NSImageView:0x7fd7bfb4b940]   (active)>",
    "<NSLayoutConstraint:0x6000031bf2a0 H:[NSImageView:0x7fd7bfb4b400]-(3)-[NSTextField:0x7fd7bfb4bbe0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf4d0 H:[NSTextField:0x7fd7bfb4bbe0]-(10)-[NSImageView:0x7fd7bfb4b6a0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf160 NSImageView:0x7fd7bfb4b400.leading == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bfac0 NSImageView:0x7fd7bfb4b940.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bf070 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bfb60 NSImageView:0x7fd7bfb4b940.width == 18   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bf1b0 NSImageView:0x7fd7bfb4b400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bf6b0 NSImageView:0x7fd7bfb4b6a0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bfa70 H:[NSImageView:0x7fd7bfb4b6a0]-(12)-[NSImageView:0x7fd7bfb4b940]   (active)>",
    "<NSLayoutConstraint:0x6000031bf2a0 H:[NSImageView:0x7fd7bfb4b400]-(3)-[NSTextField:0x7fd7bfb4bbe0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf4d0 H:[NSTextField:0x7fd7bfb4bbe0]-(10)-[NSImageView:0x7fd7bfb4b6a0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf160 NSImageView:0x7fd7bfb4b400.leading == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bfac0 NSImageView:0x7fd7bfb4b940.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bf070 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bf6b0 NSImageView:0x7fd7bfb4b6a0.width == 16   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bf1b0 NSImageView:0x7fd7bfb4b400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bf6b0 NSImageView:0x7fd7bfb4b6a0.width == 16   (active)>",
    "<NSLayoutConstraint:0x6000031bfa70 H:[NSImageView:0x7fd7bfb4b6a0]-(12)-[NSImageView:0x7fd7bfb4b940]   (active)>",
    "<NSLayoutConstraint:0x6000031bf2a0 H:[NSImageView:0x7fd7bfb4b400]-(3)-[NSTextField:0x7fd7bfb4bbe0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf4d0 H:[NSTextField:0x7fd7bfb4bbe0]-(10)-[NSImageView:0x7fd7bfb4b6a0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf160 NSImageView:0x7fd7bfb4b400.leading == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bfac0 NSImageView:0x7fd7bfb4b940.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bf070 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bf6b0 NSImageView:0x7fd7bfb4b6a0.width == 16   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bf1b0 NSImageView:0x7fd7bfb4b400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bfa70 H:[NSImageView:0x7fd7bfb4b6a0]-(12)-[NSImageView:0x7fd7bfb4b940]   (active)>",
    "<NSLayoutConstraint:0x6000031bf2a0 H:[NSImageView:0x7fd7bfb4b400]-(3)-[NSTextField:0x7fd7bfb4bbe0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf4d0 H:[NSTextField:0x7fd7bfb4bbe0]-(10)-[NSImageView:0x7fd7bfb4b6a0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf160 NSImageView:0x7fd7bfb4b400.leading == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bfac0 NSImageView:0x7fd7bfb4b940.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bf070 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bf1b0 NSImageView:0x7fd7bfb4b400.width == 15   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bf1b0 NSImageView:0x7fd7bfb4b400.width == 15   (active)>",
    "<NSLayoutConstraint:0x6000031bfa70 H:[NSImageView:0x7fd7bfb4b6a0]-(12)-[NSImageView:0x7fd7bfb4b940]   (active)>",
    "<NSLayoutConstraint:0x6000031bf2a0 H:[NSImageView:0x7fd7bfb4b400]-(3)-[NSTextField:0x7fd7bfb4bbe0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf4d0 H:[NSTextField:0x7fd7bfb4bbe0]-(10)-[NSImageView:0x7fd7bfb4b6a0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf160 NSImageView:0x7fd7bfb4b400.leading == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bfac0 NSImageView:0x7fd7bfb4b940.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bf070 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bf1b0 NSImageView:0x7fd7bfb4b400.width == 15   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bfa70 H:[NSImageView:0x7fd7bfb4b6a0]-(12)-[NSImageView:0x7fd7bfb4b940]   (active)>",
    "<NSLayoutConstraint:0x6000031bf2a0 H:[NSImageView:0x7fd7bfb4b400]-(3)-[NSTextField:0x7fd7bfb4bbe0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf4d0 H:[NSTextField:0x7fd7bfb4bbe0]-(10)-[NSImageView:0x7fd7bfb4b6a0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf160 NSImageView:0x7fd7bfb4b400.leading == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bfac0 NSImageView:0x7fd7bfb4b940.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bf070 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bfa70 H:[NSImageView:0x7fd7bfb4b6a0]-(12)-[NSImageView:0x7fd7bfb4b940]   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bfa70 H:[NSImageView:0x7fd7bfb4b6a0]-(12)-[NSImageView:0x7fd7bfb4b940]   (active)>",
    "<NSLayoutConstraint:0x6000031bf2a0 H:[NSImageView:0x7fd7bfb4b400]-(3)-[NSTextField:0x7fd7bfb4bbe0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf4d0 H:[NSTextField:0x7fd7bfb4bbe0]-(10)-[NSImageView:0x7fd7bfb4b6a0]   (active)>",
    "<NSLayoutConstraint:0x6000031bf160 NSImageView:0x7fd7bfb4b400.leading == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.leading + 10   (active)>",
    "<NSLayoutConstraint:0x6000031bfac0 NSImageView:0x7fd7bfb4b940.trailing == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.trailing - 12   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bf070 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bfa70 H:[NSImageView:0x7fd7bfb4b6a0]-(12)-[NSImageView:0x7fd7bfb4b940]   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.
Conflicting constraints detected: (
    "<NSLayoutConstraint:0x6000031bfc00 H:|-(5)-[NSVisualEffectView:0x7fd7bfb4b160](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfb4aac0 )>",
    "<NSLayoutConstraint:0x6000031bfc50 NSVisualEffectView:0x7fd7bfb4b160.right == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bf070 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.width == 0   (active)>"
).

Will attempt to recover by breaking <NSLayoutConstraint:0x6000031bfc50 NSVisualEffectView:0x7fd7bfb4b160.right == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.right - 5   (active)>.
Unable to simultaneously satisfy constraints:
(
    "<NSLayoutConstraint:0x6000031bfc00 H:|-(5)-[NSVisualEffectView:0x7fd7bfb4b160](LTR)   (active, names: '|':HeliPort.WifiMenuItemView:0x7fd7bfb4aac0 )>",
    "<NSLayoutConstraint:0x6000031bfc50 NSVisualEffectView:0x7fd7bfb4b160.right == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.right - 5   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x6000031bf070 h=--& v=--& HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x6000031bfc50 NSVisualEffectView:0x7fd7bfb4b160.right == HeliPort.WifiMenuItemView:0x7fd7bfb4aac0.right - 5   (active)>

Set the NSUserDefault NSConstraintBasedLayoutVisualizeMutuallyExclusiveConstraints to YES to have -[NSWindow visualizeConstraints:] automatically called when this happens.  And/or, set a symbolic breakpoint on LAYOUT_CONSTRAINTS_NOT_SATISFIABLE to catch this in the debugger.

Reply:
You can do
```swift
        super.init(frame: .zero)
        translatesAutoresizingMaskIntoConstraints = false
```

I tried this and it looked fine without explicit constraints here. Second line solves constraints conflict.

Reply:
Thank you, change made and confirmed that all seems good, no more hardcoded 286.0 :-)